suite/tests/client-interface/drcontainers-test.dll.c

```diff
@@ -76,10 +77,35 @@ test_vector(void)
     CHECK(ok, "drvector_delete failed");
 }
 
+unsigned int c;
+
+void count(void *payload){
```

Comment:
Type and { on separate lines, as in style guide (https://github.com/DynamoRIO/dynamorio/wiki/Code-Style)